app/models/dataset.rb

```diff
@@ -61,8 +61,7 @@ def publish!
   # What we actually want to index in Elastic, rather than the whole
   # dataset.
   def as_indexed_json(_options = {})
-    as_json(
-      only: %i[
+    as_json(only: %i[
```

Comment:
What are we gaining by changing this?

Reply:
I don't recall any style guide ever suggesting having a hanging method bracket being a good practice.

Reply:
I think it's fair to make a small improvement on the same piece of code, especially as it doesn't modify the diff. 

Perhaps we should do some pairing next week so that we understand each others' expectations better.